src/Bundle/Sender/Adapter/SwiftMailerAdapter.php

```diff
@@ -64,9 +64,11 @@ public function send(
 
         $message->setBody($renderedEmail->getBody(), 'text/html');
 
-        foreach ($attachments as $attachment) {
-            $file = \Swift_Attachment::fromPath($attachment);
-
+        foreach ($attachments as $attachmentFilename => $attachmentPath) {
+            $file = \Swift_Attachment::fromPath($attachmentPath);
+            if (!is_numeric($attachmentFilename)) {
```

Comment:
```suggestion
            if (is_string($attachmentFilename)) {
```
No need to restrict the usage of numbers as file names.